esmvaltool/cmor/tables/cmip5/Tables/CMIP5_Omon

```diff
@@ -9,13 +9,13 @@ project_id:   CMIP5  ! project id
 table_date:   17 July 2013 ! date this table was constructed
 
 missing_value: 1.e20      ! value used to indicate a missing value
-                          !   in arrays output by netCDF as 32-bit IEEE 
+                          !   in arrays output by netCDF as 32-bit IEEE
```

Comment:
Please don't change this file: if we update the cmor tables we will need to review all your changes.

Reply:
Yes, because now I need to review the whole file.